cmd/eksctl-anywhere/cmd/generateclusterconfig.go

```diff
@@ -59,7 +58,7 @@ func generateClusterConfig(clusterName string) error {
 	var machineGroupYaml [][]byte
 	var clusterConfigOpts []v1alpha1.ClusterGenerateOpt
 	switch strings.ToLower(viper.GetString("provider")) {
-	case docker.ProviderName:
```

Comment:
If the constants live in the providers then they can't be used from certain packages because they create cyclic dependencies
The other alternative is to duplicate them

What's the concern with mapping?